### PR TITLE
Add more specific description about restart and stop command

### PR DESCRIPTION
**Pull request by Zheaoli**
Signed-off-by: Zheao.Li <me@manjusaka.me>

### CODE REVIEW

**Comment by fahedouch** [comment]
LGTM on green

**Comment by AkihiroSuda** [comment]
> Signed-off-by: Manjusaka

Could you consider keeping the name consistent with your other PRs?
Real name is preferred.

**Comment by Zheaoli** [comment]
> Could you consider keeping the name consistent with your other PRs?
> Real name is preferred.

Sorry for that, updated

File: README.md [hunk 1]
```diff
@@ -706,6 +706,7 @@ Usage: `nerdctl stop [OPTIONS] CONTAINER [CONTAINER...]`
 
 Flags:
 - :whale: `-t, --time=SECONDS`: Seconds to wait for stop before killing it (default "10")
+  - Tips: If the init process in container is exited after receiving SIGTERM or exited before the time you specified, the container will be exited simultaneously
```
Comment (AkihiroSuda) [inline README.md:709]: I don't get what "simultaneously" means. Maybe s/simultaneously/immediately/ ?

Comment (Zheaoli) [reply README.md:709]: Yes! I will update the word

**Review by AkihiroSuda — APPROVED** [review]
thanks

**Pull request merged**